test/f90_correct/src/proc_ptr.f90

```diff
@@ -0,0 +1,30 @@
+!
+! Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+! See https://llvm.org/LICENSE.txt for license information.
+! SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+!
+! Test the fix for segment fault about procedure pointer under the following
+! conditions:
+! 1) module procedue used as proc-interface
+! 2) keyword RESULT appears
+! 3) none dummy argument
```

Comment:
To my understanding, your code fix is for the condition when all the three conditions are satisfied, not one of them. If my understanding is correct, the comments may need to be changed by replacing conditions with one condition.

Reply:
You are right. I rephrased the description as suggested. Thanks!